libraries/AP_Scripting/applets/video-stream-information.md

```diff
@@ -0,0 +1,31 @@
+# Video Stream Information
+
+This script updates the ArduPilot Camera library with the values required to populate the VIDEO_STREAM_INFORMATION
+mavlink message allow the GCS to find and display live video to the user
+
+# Parameters
+
+-- VID1_CAMMODEL: Video stream camera model (1:Siyi A8, 2:Siyi ZR10, 3:Siyi ZR30, 4:Siyi ZT30 Zoom, 5:Siyi ZT30 Wide, 6:Siyi ZT30 IR, 7:Siyi ZT6 RGB, 8:Siyi ZT6 IR)
+-- VID1_ID: Video stream id
+-- VID1_TYPE: Video stream type (0:RTSP, 1:RTPUDP, 2:TCP_MPEG, 3:MPEG_TS)
+-- VID1_FLAG: Video stream flags (Bitmask: 0:Running,1:Thermal,2:Thermal Range Enabled)
+-- VID1_FRAME_RATE: Video stream frame rate
+-- VID1_HRES: Video stream horizontal resolution
+-- VID1_VRES: Video stream vertical resolution
+-- VID1_BITR: Video stream bitrate
+-- VID1_HFOV: Video stream horizontal FOV in degrees
+-- VID1_ENCODING: Video stream encoding (0:Unknown, 1:H264, 2:H265)
+-- VID1_IPADDR0: Video stream IP Address first octet
+-- VID1_IPADDR1: Video stream IP Address second octet
+-- VID1_IPADDR2: Video stream IP Address third octet
+-- VID1_IPADDR3: Video stream IP Address fourth octet
+-- VID1_IPPORT: Video Stream IP Address Port
+
+# How To Use
+
+1. setup the camera gimbal as described on the ArduPilot wiki including ethernet setup
+2. check the IP address of the camera gimbal
+3. copy this script to the vehicle autopilot's "scripts" directory
+4. set the VID1_CAMMODEL parameter to the camera gimbal model
```

Comment:
```suggestion
4. setup scripting per https://ardupilot.org/plane/docs/common-lua-scripts.html and reboot . Set the VID1_CAMMODEL parameter to the camera gimbal model and adjust the new VID1 params as required.
```

Reply:
great, adjusted the how-to-use instructions, thanks!